### DISCUSSION TITLE
Error de Css en Deploy

### DISCUSSION THREAD

**Discussion by MiguelGuerrero07** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#286] [Q&A]
Hola Buenas noches.

He terminado mi practica de la semana 2 y al momento de desplegarla no carga los estilos.

Espero me puedan ayudar 
![image](https://user-images.githubusercontent.com/69058719/156972163-e0d0bea6-0f7c-41ca-8f60-7d535654a589.png)

**Comment by OscarPedroza** [comment]
Hola buenas noches, ¿Cuál es la ruta de tu archivos css en el proyecto?

**Comment by JuanSalvador73** [comment]
¿Cual es la ruta de los estilos? Lo recomendable es tener un folder raíz, donde colocas adentro el index.html y también en folders ahí mismo los de las imágenes y los estilos. Las rutas deben coincidir en tu HTML.

Si moviste los folders de su ruta con todo imágenes y estilos y estos no concuerdan en ese mismo folder, te va a marcar error, no apareciendo.
